### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by JLockerman**
Needs PR #1015 first.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,43 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.2.0 (2019-01-29)
+
+This release adds bugfixes, and optimizations such as ordered append.
```
Comment (dianasaur323) [inline CHANGELOG.md]: For this section starting from "This release and up to "Notable commits" let's go with:

**This is our first Timescale-Licensed release!**

We are excited to be introducing new time-series analytical functions, advanced data lifecycle management capabilities, and improved performance. 
- **Time-series analytical functions**: Users can now use our time_bucket_gapfill function to write complex gapfilling, last object carried forward, and interpolation queries.
- **Advanced data lifecycle management**: We are introducing scheduled policies, which use our background worker framework to manage time-series data. In this release, we support scheduled `drop_chunks` and `reorder`. 
- **Improved performance**: We added support for ordered appends, which optimize a large range of queries - particularly those that contain a GROUP BY and LIMIT clause.

This release adds code under a new license, LICENSE_TIMESCALE. This code can be found in `tsl`.

^^ maybe sanity check that I didn't fudge any facts there?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,43 @@
...
+**This is our first Enterprise release!**
+
+This release adds code under a new license, LICENSE-TIMESCALE. This code can be
+found in `tsl`, and contains scheduled tasks, such as a scheduled `drop_chunks`
+and `reorder`.
+
+**Notable commits**
+
+[fa61613]Change time_bucket_gapfill argument names
+[fdaa717] Update telemetry with prettier os info
```
Comment (dianasaur323) [inline CHANGELOG.md]: I think we can remove this

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,43 @@
...
+[01be394] Fix bgw_launcher restart when failing during launcher setup
+[7b3929e] Add ordered append optimization
+[a69f84c] Fix signal processing in background workers
+[65894f0] Add view displaying info about the current license
```
Comment (dianasaur323) [inline CHANGELOG.md]: I think remove

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,43 @@
...
+[47b5b7d] Log which chunks are dropped by background workers
+[057eef1] Log to pg log when running reorder jobs
```
Comment (dianasaur323) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,43 @@
...
+[ea718e0] Make older_than / newer_than error message user friendly
```
Comment (dianasaur323) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,43 @@
...
+[8b99d91] Add telemetry reports for timescaledb.last_tuned and last_tuned_version
```
Comment (dianasaur323) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,43 @@
...
+[4e1e15f] Add reorder command
+[2e4bb5d] Recluster and drop chunks scheduling code
+[ef43e52] Add alter_policy_schedule API function
+[5ba740e] Add gapfill query support
+[be7c74c] Add logic for automatic DB maintenance functions
+[568749c] Add more license key fields and telemetry
```
Comment (dianasaur323) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md (hunk 2 above)
Comment (creatorrr) [reply CHANGELOG.md]: Also need to convert this section into a markdown list so it renders correctly

File: CHANGELOG.md (hunk 1 above)
Comment (mfreed) [reply CHANGELOG.md]: "This is our first release to include Timescale-Licensed features, in addition to new Apache-2 capabilities."

And put `time_bucket_gapfill` in backticks

File: CHANGELOG.md (hunk 2 above)
Comment (JLockerman) [reply CHANGELOG.md]: I always forget that.

**Comment by JLockerman** [comment]
@RobAtticus didn't see anything in the commits when I looked over them, but I may have missed something.

File: CHANGELOG.md (hunk 1 above)
Comment (RobAtticus) [reply CHANGELOG.md]: The first line `This release adds bugfixes...` should be cut.

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,39 @@
...
+* [5ba740e] Add gapfill query support
+* [be7c74c] Add logic for automatic DB maintenance functions
+* [4ff6ac7] Initial Timescale-Licensed-Module and License-Key Implementation
+* [fc42539] Add new top-level licensing information
+* [31e9c5b] Fix time column handling in get_create_command
+* [1b8ceca] Avoid loading twice in parallel workers and load only from $libdir
+* [76d7875] Don't throw errors when extension is loaded but not installed yet
+* [eecd845] Add Timescale License (TSL)
+* [4b42b30] Free ChunkInsertStates when the es_per_tuple_exprcontext is freed
+* [43e2021] Set C standard to C11
```
Comment (RobAtticus) [inline CHANGELOG.md]: Cut?

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,47 @@
...
+* [fc42539] Add new top-level licensing information
+* [31e9c5b] Fix time column handling in get_create_command
+* [1b8ceca] Avoid loading twice in parallel workers and load only from $libdir
+* [76d7875] Don't throw errors when extension is loaded but not installed yet
+* [eecd845] Add Timescale License (TSL)
+* [4b42b30] Free ChunkInsertStates when the es_per_tuple_exprcontext is freed
+
+**Thanks**
+
+* Asbjorn
```
Comment (RobAtticus) [inline CHANGELOG.md]: Accident?

File: CHANGELOG.md [hunk 10]
```diff
@@ -4,6 +4,47 @@
...
+* @fordred for reporting our docker-run.sh script was out of date
+* @JpWebster for reporting a deadlock between reads an drop_chunks
+* @chickenburgers for reporting an issue with our CMake
+* Dimtrj and Asbjørn D. for creating a reproducible testcase for an UPSERT bug
```
Comment (RobAtticus) [inline CHANGELOG.md]: Are these Github names...or?

File: CHANGELOG.md (hunk 9 above)
Comment (JLockerman) [reply CHANGELOG.md]: yup

File: CHANGELOG.md (hunk 10 above)
Comment (JLockerman) [reply CHANGELOG.md]: nope, slack, mentioning that

**Pull request merged**